Source/Krypton Components/Krypton.Toolkit/Palette Builtin/Base/PaletteBase.cs

```diff
@@ -26,7 +26,7 @@ public abstract class PaletteBase : Component
         private Padding? _inputControlPadding;
         private PaletteDragFeedback _dragFeedback;
         private string _themeName;
-        private Image[] /*ICollection<Image?>*/ _buttonSpecImages;
+        protected static Image[] /*ICollection<Image?>*/ _buttonSpecImages;
```

Comment:
What happens when a new palette is loaded, how will these images be replaced ?
i.e. Not sure static is the right option.

Reply:
Yes, I noticed while testing today that it doesn't work. Going to revert.

Reply:
Threw a assertion